test/sequential/test-timers-blocking-callback.js

```diff
@@ -28,17 +29,26 @@ const Timer = process.binding('timer_wrap').Timer;
 
 const TIMEOUT = 100;
 
-let nbBlockingCallbackCalls = 0;
-let latestDelay = 0;
-let timeCallbackScheduled = 0;
+let nbBlockingCallbackCalls;
+let latestDelay;
+let timeCallbackScheduled;
+
+// These tests are somewhat probablistic so they may fail even when the bug is
```

Comment:
@misterdjules I updated the comment to remove the term "probablistic" and to hopefully add a bit more clarity about what the problem is.